src/forms/FormFieldDropdownSingleSelection/FormFieldDropdownSingleSelection.tsx

```diff
@@ -62,6 +70,7 @@ const DropdownSingleSelection = (props: MosaicFieldProps<DropdownSingleSelection
 						data-testid="autocomplete-test-id"
 						options={fieldDef?.inputSettings?.options}
 						getOptionLabel={(option) => option?.label ? option.label : ""}
+						getOptionSelected={getOptionSelected}

```

Comment:
Not completely sure if this has something to do with this but when I open the storybook, all options appear like this 
![image](https://user-images.githubusercontent.com/23409284/167041793-8ac3320e-ebc8-43c4-8634-b75e45105040.png)


This is how it looks on master:
![image](https://user-images.githubusercontent.com/23409284/167041949-ac07e7ad-96a5-48ea-95cb-0cadb3b7dfb2.png)